pyproject.toml

```diff
@@ -104,13 +108,20 @@ pycryptodome = "~3.10.1"
 pyuwsgi = "~2.0.19.1.post0"
 # YAML parsing and rendering
 PyYAML = "~5.4.1"
+# Social authentication core
+social-auth-core = {extras = ["openidconnect", "saml"], version = "~4.1.0", optional = true}
```

Comment:
Does marking social-auth-core as optional actually make sense given that social-auth-app-django is non-optional and it in turn requires social-auth-core? Is this just for the optional openidconnect and saml extras? I ask because the `poetry.lock` here looks suspicious (https://github.com/nautobot/nautobot/pull/1073/files#diff-f53a023eedfa3fbf2925ec7dc76eecdc954ea94b7e47065393dbad519613dc89R1731) - note the empty dependencies for `openid` and `saml`:

```
[extras]
all = ["mysqlclient", "django-auth-ldap", "django-storages"]
ldap = ["django-auth-ldap"]
mysql = ["mysqlclient"]
openid = []
remote_storage = ["django-storages"]
saml = []
```